pyk4a/capture.py

```diff
@@ -50,6 +53,24 @@ def color_timestamp_usec(self) -> int:
             self.color
         return self._color_timestamp_usec
 
+    @property
+    def color_exposure_usec(self) -> int:
```

Comment:
Before we make one function for each, should we fetch all of the properties/metadata at the same time, in a single function?

Reply:
If those two are the only available properties, let's keep it that way.